React/Views/RCTScrollView.m

```diff
@@ -419,24 +425,32 @@ - (void)setRemoveClippedSubviews:(__unused BOOL)removeClippedSubviews
 - (void)insertReactSubview:(UIView *)view atIndex:(NSInteger)atIndex
 {
   [super insertReactSubview:view atIndex:atIndex];
+#if !TARGET_OS_TV
   if ([view isKindOfClass:[RCTRefreshControl class]]) {
     [_scrollView setRctRefreshControl:(RCTRefreshControl *)view];
   } else {
+#endif
```

Comment:
One thing you can do to make the `#if` a bit less frequent is structure it like this:

```objective-c
#if !TARGET_OS_TV
if ([view isKindOfClass:[RCTRefreshControl class]]) {
  [_scrollView setRctRefreshControl:(RCTRefreshControl *)view];
} else
#endif
{
  ..
}